tests/Fixtures/Integration/set/@PER-CS2.0.test-out.php

```diff
@@ -56,3 +56,22 @@ class Aaa implements
     'foo' => 'bar',
     'foo2' => 'bar',
 ];
+
+$arrayMultilineWithoutComma = [
+    'foo' => 'bar',
+    'foo2' => 'bar',
+];
+$heredocMultilineWithoutComma = [
+    'foo',
+    <<<EOD
+        bar
+        EOD,
+];
+argumentsMultilineWithoutComma(
+    1,
+    2,
+);
+function parametersMultilineWithoutComma(
+    $x,
+    $y,
```

Comment:
This will fail on PHP 7.4 because trailing comma is not allowed here. You need to duplicate this test so that one runs on PHP 7.4 (no comma added here) and the other on PHP 8.0+ (comma added). You can have a look at `tests/Fixtures/Integration/set/@PHP74Migration.test` for an example.